index.html

```diff
@@ -14,7 +14,46 @@
   <header class="header-container">
     <a class="mylogo" href="#">My Logo</a>
     <nav class="menu">
-      <img src="image/menu.png" alt="Menu Icon">
+      <img src="icons/menu.png" alt="Menu Icon">
     </nav>
   </header>
-</body>
\ No newline at end of file
+  <main class="home">
+    <section class="bio">
+      <h1 class="title">I’m Jawad <br>
+        Glad to see you!</h1>
+      <p class="text">
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <h3 class="connect">LET'S CONNECT</h3>
+        <ul>
```

Comment:
- [ ] Kindly 😃 try to put the text ```LET'S CONNECT```in the correct position as you see here the text is behind the paragraph, so please try to put this text below the paragraph to be like the Figma Design 😊
- [ ] Also consider trying to put the ``` LET'S CONNECT``` in the link a or button tag this will be good for semantics reasons , so kindly try to replace the h3 tag by link or button 💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/220177755-98e8bc20-796b-4b7c-9915-261f45b7ef43.png)  |   ![image](https://user-images.githubusercontent.com/83360636/220177700-075af61d-f7e1-4401-97d2-a8fc5f7bec9f.png) |

Reply:
Hi @osamaashraf6 

If you have a look on my code inside index.html file you will notice that there is no such a sentence in my code and I'm wired that from where you found this. May I ask you to review and check the preview of my code in another way?

![image](https://user-images.githubusercontent.com/115032263/220185760-daa1876b-bac3-46b3-9880-115e35488bb3.png)